.circleci/config.yml

```
@@ -64,6 +64,10 @@ jobs:
           name: Start OTel Collector
           command: touch ./smoke-tests/collector/data.json && ./otelcol --config=./smoke-tests/collector/otel-collector-config.yaml --set="exporters::file::path=./smoke-tests/collector/data.json"
           background: true
+      - run:
+          name: Start Mock Server
+          command: cd smoke-tests/mock-server && npm install && npm start
+          background: true
```

you could also set working directory for this I think if you wanted

```suggestion
      - run:
          name: Start Mock Server
          working_directory: smoke-tests/mock-server
          command: npm install && npm start
          background: true
```